src/pybind/mgr/dashboard_v2/controllers/task.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from ..tools import ApiController, AuthRequired, RESTController, TaskManager
+
+
+@ApiController('task')
+@AuthRequired()
+class TaskController(RESTController):
+    def list(self, namespace=None):
+        executing_t, finished_t = TaskManager.list_serialized(namespace)
+        return {
+            'executing_tasks': executing_t,
+            'finished_tasks': finished_t
+        }
```

Comment:
I'd unify both lists into one and add a `state` property.

Reply:
I can imagine this being useful from the UI point-of-view. Maybe a frontend dev could opinionate on what they would like to be returned here?